src/mount.ts

```diff
@@ -236,7 +236,7 @@ export function mount(
 
         if (typeof slot === 'string') {
           // slot is most probably a scoped slot string or a plain string
-          acc[name] = (props: VNodeProps) => h(processSlot(slot), props)
+          acc[name] = (props: VNodeProps) => h(processSlot(slot.trim()), props)
```

Comment:
nit: maybe `trim` should be called in `processSlot` (as if someone uses it in an other place of VTU, we don't have the same issue another time)

Reply:
@cexbrayat I am curious too... I had a little look but it wasn't obvious why this was causing and issue. I guess no one has tried calling `compile` where the first character is a carriage return 🤔 

I wonder if this should be fixed in core.

Reply:
Ohh I bet this is the issue https://github.com/vuejs/vue-test-utils-next/blob/e92bdcb430b6098204d20c5ea3b9c80bd21ddba7/src/utils/compileSlots.ts#L5

`startsWith('<template')` is not the same as `startsWith('\ntemplate')`.

Reply:
I made your suggested change! thanks.